src/OpenTelemetry.Instrumentation.AspNetCore/CHANGELOG.md

```diff
@@ -3,6 +3,12 @@
 ## Unreleased
 
 * Introduced support for Grpc.AspNetCore (#803).
+  * Attributes are added to gRPC invocations: `rpc.system`, `rpc.service`,
+    `rpc.method`. These attributes are are added to an existing span generated
+    by the instrumentation. This is unlike the instrumentation for client-side
+    gRPC calls where one span is created for the gRPC call and a separate span
+    is created for the underlying HTTP call in the event both gRPC and HTTP
+    instrumentation is enabled.
```

Comment:
Maybe we should switch from is enabled to "are enabled". I'm not sure (english isn't my first language)...

Reply:
Arrrr! Your English is perfect.